x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_page.tsx

```diff
@@ -18,6 +19,10 @@ import { HelpMenu } from '../components/help_menu';
 import { useMlKibana } from '../contexts/kibana';
 import { MlPageHeader } from '../components/page_header';
 
+const singleMetricViewerTitle = css({
+  minWidth: '350px',
```

Comment:
All other pages are `300px` is this larger size to account for wider text?
If so, this text will be translated and could be any size.